src/inmanta/data/__init__.py

```diff
@@ -2644,6 +2647,29 @@ def to_dto(self) -> m.Environment:
             "Default=336 hours (2 weeks). Set to 0 to disable automatic cleanups.",
             validator=convert_int,
         ),
+        SCHEDULER_DB_CONNECTION_POOL_MIN_SIZE: Setting(
+            name=SCHEDULER_DB_CONNECTION_POOL_MIN_SIZE,
+            typ="int",
+            default=1,
+            doc="The database connection pool will be initialized with this number of connections for"
+            "the resource scheduler in this environment.",
+            validator=convert_int,
+        ),
+        SCHEDULER_DB_CONNECTION_POOL_MAX_SIZE: Setting(
+            name=SCHEDULER_DB_CONNECTION_POOL_MAX_SIZE,
+            typ="int",
+            default=5,
+            doc="Limit the size of the database connection pool to this number of connections for"
+            "the resource scheduler in this environment.",
+            validator=convert_int,
+        ),
+        SCHEDULER_DB_CONNECTION_TIMEOUT: Setting(
+            name=SCHEDULER_DB_CONNECTION_TIMEOUT,
+            typ="positive_float",
+            default=60.0,
+            doc="Database connection timeout in seconds.",
+            validator=convert_positive_float,
+        ),
```

Comment:
I don't think we want to make this into environment settings. 

typically, the people with access to the DB setup have no access to the environment setting, and vice versa.

(also, setting these requires accessing the DB, so this is very risky business)